Tests/js/views/ez-testonearticlesview.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ Test One Articles view tests</title>
+</head>
+<body>
+
+<script type="text/x-handlebars-template" id="testoneview-ez-template">template based view</script>
+
+<script type="text/javascript" src="../../../Resources/public/vendors/yui3/build/yui/yui.js"></script>
+<script type="text/javascript" src="./assets/ez-testonearticlesview-tests.js"></script>
+<script>
+    var filter = (window.location.search.match(/[?&]filter=([^&]+)/) || [])[1] || 'raw',
+        loaderFilter;
+    if (filter == 'coverage'){
+        loaderFilter = {
+            searchExp : "/Resources/public/js/",
+            replaceStr: "/Tests/instrument/Resources/public/js/"
+        };
+    } else {
+        loaderFilter = filter;
+    }
+
+    YUI({
+        coverage: ['ez-testonearticlesview'],
+        filter: loaderFilter,
+        modules: {
+            "ez-testonearticlesview": {
+                requires: ['ez-view', 'handlebars', 'template', 'ez-templatebasedview'],
```

Comment:
Jeśli wymagasz aby `ez-templatebaseview` było załadowane, to nie musisz już dopisywać `'ez-view', 'handlebars', 'template'` do listy zależności, bo `ez-templatebaseview` ma je zdefiniowane w swoich zależnościach. Nie ma potrzeby duplikowania zależności.